build/cqs.tsv

```diff
@@ -1,3 +1,4 @@
+asm:asm	3.1	compile
```

Comment:
I know you may have just re-run a script to generate this.  Did you need the change in version of ASM?